docs/platform/concepts/managed-users.md

```diff
@@ -4,10 +4,20 @@ title: Managed users
 
 import ActionsIcon from "@site/static/images/icons/more.svg";
 
-The managed users feature lets you centrally manage your organization's users, including editing their profiles, resetting passwords, and [setting authentication policies](/docs/platform/howto/set-authentication-policies).
+The managed users feature lets you centrally manage your organization's users and helps you to secure your organization in Aiven.
 
-A managed user cannot create new organizations
-unless they are a [super admin](/docs/platform/howto/make-super-admin) of the organization.
+With managed users, you can:
+
+- Control how users log in with [
+  authentication policies](/docs/platform/howto/set-authentication-policies), not just how
+  they access the organization
```

Comment:
Nitpick to keep link markup "together" to help with search/replace operations that will surely happen one day.

```suggestion
- Control how users log in with [authentication policies](/docs/platform/howto/set-authentication-policies),
  not just how they access the organization
```